src/ts/components/seekbar.ts

```diff
@@ -30,6 +31,12 @@ export interface SeekBarConfig extends ComponentConfig {
    * Default: 50 (50ms = 20fps).
    */
   smoothPlaybackPositionUpdateIntervalMs?: number;
+
+  /**
+   * WCAG20 standard - denotes which elements in a page an interactive element or set of elements has control
+   * over and affects them.
+   */
+  'aria-controls'?: string;
```

Comment:
Why do we change the naming of the property here? Our style would be `ariaControls`